packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -69,6 +69,12 @@ type AlertProps = {
    */
   onDismiss?: () => void;
 
+  /**
+   * Can be used to render an icon
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  leading?: ComponentType<any>;
```

Comment:
I think it should be called `icon` similar to how we have in Button and Link? here we don't have anything in trailing